const_acc_x:
    kalman:
        noise_pos_mv: 10.5
        noise_acc_mv: 1
        noise_pos_proc: 2.75
        noise_vel_proc: 18.75
        noise_acc_proc: 0.1

const_acc_y:
    kalman:
        noise_pos_mv: 10.5
        noise_acc_mv: 1
        noise_pos_proc: 2.75
        noise_vel_proc: 18.75
        noise_acc_proc: 0.1

const_acc_z:
    kalman:
        noise_pos_mv: 6.75
        noise_acc_mv: 1
        noise_pos_proc: 2.75
        noise_vel_proc: 7
        noise_acc_proc: 0.1
